pcbnew2boardview.py

```diff
@@ -139,17 +139,40 @@ def convert(pcb, brd):
     brd.write("\n")
 
 
+def convert_obdata(pcb, obdata):
+    obdata.write("COMPONENTS_DATA_START\n### Component Category Value Comment\n### v = value, p = package, c = manufacturer code, r = rating, m = misc, s = status\n###\n")
```

Comment:
Please use a single `write()` call for each line.

Reply:
I believe I have resolved all of the requested changes, though to avoid redundancy, I will only be commenting on this review. Please let me know if anything could still use further work.